tools/const_finder/lib/const_finder.dart

```diff
@@ -58,6 +85,18 @@ class _ConstVisitor extends RecursiveVisitor<void> {
   // Avoid visiting the same constant more than once.
   final Set<Constant> _cache = LinkedHashSet<Constant>.identity();
 
+  @override
+  void visitProcedure(Procedure node) {
+    final bool isTearOff = node.isStatic &&
+        node.kind == ProcedureKind.Method &&
+        node.name.text == '_#new#tearOff';
```

Comment:
Will one of the tests below break if the compiler starts using a different name for this?

Reply:
Yes, the web tests will pick up new non-const locations not in the expectations